Explorer/Assets/Scripts/Global/Dynamic/PortableExperiences/PortableExperiencesController.cs

```diff
@@ -0,0 +1,180 @@
+        private readonly ServerAbout serverAbout = new ();
+        public Dictionary<ENS, Entity> PortableExperienceEntities { get; } = new ();
+        private World world => globalWorldProxy.Object;
+
+        public PortableExperiencesController(
+            ObjectProxy<World> world,
+            IWeb3IdentityCache web3IdentityCache,
+            IWebRequestController webRequestController,
+            IScenesCache scenesCache)
+        {
+            globalWorldProxy = world;
+            this.web3IdentityCache = web3IdentityCache;
+            this.webRequestController = webRequestController;
+            this.scenesCache = scenesCache;
+        }
+
+        public async UniTask<IPortableExperiencesController.SpawnResponse> CreatePortableExperienceByEnsAsync(ENS ens, CancellationToken ct, bool isGlobalPortableExperience = false)
+        {
+            string worldUrl = string.Empty;
+
+            if (ens.IsValid) { worldUrl = ENSUtils.ConvertEnsToWorldUrl(ens); }
+
+            if (!worldUrl.IsValidUrl()) throw new ArgumentException("Invalid Spawn params. Provide a valid ENS name");
+
+            var portableExperiencePath = URLDomain.FromString(worldUrl);
+            URLAddress url = portableExperiencePath.Append(new URLPath("/about"));
+
+            GenericDownloadHandlerUtils.Adapter<GenericGetRequest, GenericGetArguments> genericGetRequest = webRequestController.GetAsync(new CommonArguments(url), ct, ReportCategory.REALM);
+
+            ServerAbout result = await genericGetRequest.OverwriteFromJsonAsync(serverAbout, WRJsonParser.Unity);
+
+            if (result.configurations.scenesUrn.Count == 0)
+            {
+                //The loaded realm does not have any fixed scene, so it cannot be loaded as a Portable Experience
+                throw new Exception($"Scene not Available in provided Portable Experience with ens: {ens}");
+            }
+            var realmData = new RealmData();
+            realmData.Reconfigure(
+                new IpfsRealm(web3IdentityCache, webRequestController, portableExperiencePath, result),
+                result.configurations.realmName.EnsureNotNull("Realm name not found"),
+                result.configurations.networkId,
+                result.comms?.adapter ?? string.Empty,
+                result.comms?.protocol ?? string.Empty,
+                portableExperiencePath.Value
+            );
+
+            ISceneFacade parentScene = scenesCache.Scenes.FirstOrDefault(s => s.SceneStateProvider.IsCurrent);
+            string parentSceneName = parentScene != null ? parentScene.Info.Name : "main";
+            Entity portableExperienceEntity = world.Create(new PortableExperienceRealmComponent(realmData, parentSceneName, isGlobalPortableExperience), new PortableExperienceComponent(ens));
+            PortableExperienceEntities.Add(ens, portableExperienceEntity);
+
+            WaitToKill(ens).Forget();
```

Comment:
There is something not right: you call to kill the portable experience right after you added it, was it for test only?

Reply:
Yep :S